### PR TITLE
Update health-content.md

### PR DESCRIPTION
**Pull request by megzehn**
updating the health-content section in the style guide per ticket [81486](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81486)

### CODE REVIEW

**Comment by megzehn** [comment]
Hi @humancompanion-usds, I updated a page in our style guide based on [these edits](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81486) from @DanielleThierryUSDSVA and @laurwill. Could you review and merge when you get a chance?

One thing: I changed the headers in the Like this / Not this so that they kept the correct hierarchy. Could you review and make sure I didn't mess up the styling? Thank you so much!

File: src/_content-style-guide/health-content.md [hunk 1]
```diff
@@ -5,219 +5,229 @@ title: Health content
...
+Many Veterans live with chronic (long-lasting) diseases—like diabetes and high blood pressure—that often require behavior changes as part of a treatment plan. Follow these tips for content that promotes health behavior change.
+
+### Give specific actions—and emphasize the benefits
+
+When writing health-related instructions, provide small and specific action steps. Use everyday examples. And remind people how taking these actions can help them.
 
 <div class="do-dont">
 <div class="do-dont__do">
-<h3 class="do-dont__heading">Like this</h3>
+<h3 class="do-dont__heading">Like this</h4>
```
Comment (humancompanion-usds) [inline src/_content-style-guide/health-content.md:83]: This starts with an h3 but ends with an h4. 
```suggestion
<h4 class="do-dont__heading">Like this</h4>
```

File: src/_content-style-guide/health-content.md [hunk 2]
```diff
@@ -5,219 +5,229 @@ title: Health content
...
+- A large bag of garbage
+- A laundry basket filled with towels or jeans
+- A large bag of sugar or flour, or a sack of potatoes
 
 </div>
 </div>
 
 <div class="do-dont__dont">
-<h3 class="do-dont__heading">Not this</h3>
+<h3 class="do-dont__heading">Not this</h4>
```
Comment (humancompanion-usds) [inline src/_content-style-guide/health-content.md:101]:
```suggestion
<h4 class="do-dont__heading">Not this</h4>
```

File: src/_content-style-guide/health-content.md [hunk 3]
```diff
@@ -5,219 +5,229 @@ title: Health content
...
-#### Example 1
+### Be practical and realistic
 
-When directing Veterans to crisis counseling, frame the language around the specific benefit (talking to someone right away for support). By reframing the language, we don't force the Veteran to question if they fit into the definition of being "in crisis."
+Consider how factors beyond Veterans’ control affect their ability to follow health recommendations. Avoid describing a behavior change as easy or simple when it may be hard to do in practice.
 
 <div class="do-dont">
 <div class="do-dont__do">
-<h3 class="do-dont__heading">Like this</h3>
+<h3 class="do-dont__heading">Like this</h4>
```
Comment (humancompanion-usds) [inline src/_content-style-guide/health-content.md:117]:
```suggestion
<h4 class="do-dont__heading">Like this</h4>
```

File: src/_content-style-guide/health-content.md [hunk 4]
```diff
@@ -5,219 +5,229 @@ title: Health content
...
-
-Whatever you're struggling with, our responders can offer confidential help 24/7. Many of them are Veterans themselves.
+Eating healthy foods can help you feel better. Try to plan healthy meals that fit your budget. If fresh produce is too expensive, try frozen fruits and veggies instead.
 
 </div>
 </div>
 
 <div class="do-dont__dont">
-<h3 class="do-dont__heading">Not this</h3>
+<h3 class="do-dont__heading">Not this</h4>
```
Comment (humancompanion-usds) [inline src/_content-style-guide/health-content.md:126]:
```suggestion
<h4 class="do-dont__heading">Not this</h4>
```

File: src/_content-style-guide/health-content.md [hunk 5]
```diff
@@ -5,219 +5,229 @@ title: Health content
...
+Many Veterans may have a higher risk of certain conditions based on their service history. When writing about risk, make the numbers as clear and simple as possible. And always follow statements about risk with action steps your audience can take to protect their health
+
+### Do the math for your audience
+
+Use comparative numbers (like “1 in” or “1 out of”) language instead of fractions or percentages. If you’re comparing numbers, always keep the denominator the same.
 
 <div class="do-dont">
 <div class="do-dont__do">
-<h3 class="do-dont__heading">Like this</h3>
+<h3 class="do-dont__heading">Like this</h4>
```
Comment (humancompanion-usds) [inline src/_content-style-guide/health-content.md:171]:
```suggestion
<h4 class="do-dont__heading">Like this</h4>
```

File: src/_content-style-guide/health-content.md [hunk 6]
```diff
@@ -5,219 +5,229 @@ title: Health content
...
 
-Using your asthma inhaler correctly will help you breathe easier. Follow the directions for your inhaler. If you’re unsure of anything, ask your health care provider to explain.
+About 8 out of 10 people reported side effects from the older treatment. But only 2 out of 10 people reported side effects from this newer treatment.
 
 </div>
 </div>
+
 <div class="do-dont__dont">
-<h3 class="do-dont__heading">Not this</h3>
+<h3 class="do-dont__heading">Not this</h4>
```
Comment (humancompanion-usds) [inline src/_content-style-guide/health-content.md:180]:
```suggestion
<h4 class="do-dont__heading">Not this</h4>
```

File: src/_content-style-guide/health-content.md [hunk 7]
```diff
@@ -5,219 +5,229 @@ title: Health content
...
-- A large watermelon
-- A large bag of sugar or flour, or a sack of potatoes
+Eating a lot of processed meat may increase your risk of getting colorectal cancer. But the increase is relatively small. Studies show that eating 50 grams of processed meat every day can increase your lifetime risk from about 5 in 100 to about 6 in 100.
 
 </div>
 </div>
+
 <div class="do-dont__dont">
-<h3 class="do-dont__heading">Not this</h3>
+<h3 class="do-dont__heading">Not this</h4>
```
Comment (humancompanion-usds) [inline src/_content-style-guide/health-content.md:203]:
```suggestion
<h4 class="do-dont__heading">Not this</h4>
```

**Review by humancompanion-usds — APPROVED** [review]
I fixed up the headers.

**Comment by laurwill** [comment]
Hi @humancompanion-usds , looks like something funky happened with the formatting. Do you have a quick fix for this, or could you revert while we figure it out?

image

**Pull request merged**